python/lsst/ctrl/mpexec/singleQuantumExecutor.py

```diff
@@ -27,15 +27,23 @@
 from collections import defaultdict
 import logging
 from itertools import chain
+import sys
 import time
 
 # -----------------------------
 #  Imports for other modules --
 # -----------------------------
 from .quantumGraphExecutor import QuantumExecutor
 from lsst.log import Log
+from lsst.daf.base import PropertyList, PropertySet
```

Comment:
Why are we now adding a dependency on daf_base when we are trying to get gen3 middleware to stop using it?

Reply:
Because tasks do still use it, and it's not hard to back out later.